server/Arcadia.Assistant.Web/Employees/PendingActionsRequest.cs

```diff
@@ -157,6 +154,25 @@ void OnMessage(object message)
 
                         break;
 
+                    case GetEmployeePendingActions.Response response:
+                        var pendingActions = response.PendingActions.ToList();
+                        if (pendingActions.Count != 0)
+                        {
+                            var groupedActions = pendingActions.GroupBy(a => a.EmployeeId);
+                            foreach (var grouping in groupedActions)
+                            {
+                                eventsByEmployeeId.TryAdd(grouping.Key, new List<CalendarEvent>());
+                                eventsByEmployeeId[grouping.Key].AddRange(grouping);
+                            }
+                        }
+
+                        calendarActorsToRespond.Remove(this.Sender);
```

Comment:
we won't need this actor, right?

Reply:
Not right :) For now, only vacations are added to specific pending actions actor, because only them follow specific pipeline (create -> ask for approve (add to pendings of concrete person) -> approve -> ask one more approve (add to pendings of another person)). All other event types don't know which person they should be shown to.